components/web/apps/org.wso2.carbon.apimgt.store/src/main/fragments/composite-api-implementation/composite-api-implementation.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="add-padding-4x">
+    <div class="form-horizontal">
+        <div class="form-group">
+            <p>Goto Ballerina editor inorder to implement your composition logic</p>
```

Comment:
Shall we correct the typo here? 'Go to the Ballerina editor in order to implement your composition logic'